CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### 💡 Enhancements 💡
 
 - `collector, target allocator, opamp`: Require Go 1.22 for building the operator (#2757)
+- `autoinstrumentation`: Python Automatic Instrumentation support for prometheus exporter (#3122)
```

Comment:
It does not belong here please create new changelog entry with `make chlog-new`

Reply:
sorry for that! I corrected it.